tests/data_context/cloud_data_context/test_expectation_suite_crud.py

```diff
@@ -607,3 +611,72 @@ def test_add_or_update_expectation_suite_updates_existing_obj(
         context.add_or_update_expectation_suite(expectation_suite=suite)
 
     mock_update.assert_called_once()
+
+
+@pytest.mark.integration
+def test_get_expectation_suite_include_rendered_content_prescriptive(
+    empty_data_context,
+):
+    context = empty_data_context
+
+    expectation_suite_name = "validating_taxi_data"
+
+    expectation_configuration = ExpectationConfiguration(
+        expectation_type="expect_column_max_to_be_between",
+        kwargs={
+            "column": "passenger_count",
+            "min_value": {"$PARAMETER": "upstream_column_min"},
+            "max_value": {"$PARAMETER": "upstream_column_max"},
+        },
+    )
+
+    context.add_expectation_suite(
+        expectation_suite_name=expectation_suite_name,
+        expectations=[expectation_configuration],
+    )
+
+    expectation_suite_exclude_rendered_content: ExpectationSuite = (
+        context.get_expectation_suite(
+            expectation_suite_name=expectation_suite_name,
+        )
+    )
+    assert (
+        expectation_suite_exclude_rendered_content.expectations[0].rendered_content
+        is None
+    )
+
+    expected_expectation_configuration_prescriptive_rendered_content = [
+        RenderedAtomicContent(
+            value_type="StringValueType",
+            value=RenderedAtomicValue(
+                schema={"type": "com.superconductive.rendered.string"},
+                template="$column maximum value must be greater than or equal to $min_value and less than or equal to $max_value.",
+                params={
+                    "column": {
+                        "schema": {"type": "string"},
+                        "value": "passenger_count",
+                    },
+                    "min_value": {
+                        "schema": {"type": "object"},
+                        "value": {"$PARAMETER": "upstream_column_min"},
+                    },
+                    "max_value": {
+                        "schema": {"type": "object"},
+                        "value": {"$PARAMETER": "upstream_column_max"},
+                    },
+                },
+            ),
```

Comment:
@josectobar this is the rendering the was failing before (we weren't testing it).